crates/bevy_ecs/src/query/access.rs

```diff
@@ -306,6 +354,20 @@ impl<T: SparseSetIndex> FilteredAccessSet<T> {
         true
     }
 
+    /// Returns `true` if this and `filtered_access` can be active at the same time.
+    pub fn is_compatible_single(&self, filtered_access: &FilteredAccess<T>) -> bool {
```

Comment:
Don't love this name; I have a hard time reasoning about what it might do.

Reply:
Oh I see, it's analogous to `get_conflicts_single`. Okay, fine for now.

Reply:
Yep, since I undid my `Access` trick, there is no component representing the world's metadata, so  `get_conflicts` can return an empty vector while `is_compatible()` can still say that it's `false` for any pair involving an exclusive system.